backend/curation/api/v1/curation/collections/actions.py

```diff
@@ -66,8 +67,11 @@ def post(body: dict, user: str):
         body.get("consortia", []),
     )
 
+    # Get 'curator_name' from access token
+    curator_name = get_payload_from_request_context()["curator_name"]
+
     try:
-        version = get_business_logic().create_collection(user, "", metadata)
+        version = get_business_logic().create_collection(user, curator_name, metadata)
```

Comment:
this can be simplified by adding `token_info: dict` as a parameter to the post. [Connexion magic](https://connexion.readthedocs.io/en/latest/security.html) will pass the decoded token_info into the function. Then you wont have to store the token_info in `flask.g`

Reply:
@Bento007 this is amazing. And it makes tons of sense. Everything I had found just mentioned strict passing of the str var through. Thanks!! Will make the change.